_docs/_latest/configure/notifier/index.md

```diff
@@ -29,6 +29,63 @@ and varying formats to you to events in your Google Cloud Platform
 
 ## Configuring Notifier
 
+### Email Connector Config
+
+Forseti security provides an interface to add the email connector of your
+choice. A new section named `email_connector` has been introduced under 
+`notifier` as part of the new design. The connector details provided in this
+section will be used to send out inventory summary and violations via email
+when found.notify users violations when found for the enabled
+scanners. 
+
+To configure `email_connector`, follow the steps below:
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+2. Navigate to the `notifier` > `email_connector` section.
+
+If you want the notifier to send violations and/or inventory summary via email, 
+provide valid values for all the fields mentioned below.
+
+* `auth`
+  * **Description**: The connector you want to use to receive emails.
+  * **Valid values**: String
+  
+* `auth`
+  * **Description**: The authentication/authorization details required key used to authorize requests to SendGrid.
+  * **Valid values**: String
+
+* `sender`
+  * **Description**: The email address of the sender of the email.
+  * **Valid values**: String
+
+* `recipient`
+  * **Description**: The email addresses of the recipients of the email.
+  * **Valid values**: String
+  * **Note**: Multiple email recipients as delimited by comma, like
```

Comment:
Multiple email addresses delimited by comma, for example

Reply:
Fixed